lib/slack-rtmapi/client.rb

```diff
@@ -97,6 +105,7 @@ def main_loop
       init
       loop do
         inner_loop
+        sleep 0.5
```

Comment:
this shouldn't be necessary as the reading is already blocking. Why did you need to add that ? What was the issue ? It could be a hidden bug